lib/common_test/src/ct.erl

```diff
@@ -975,7 +976,20 @@ make_priv_dir() ->
 %%% belongs to.
 get_target_name(Handle) ->
     ct_util:get_target_name(Handle).
-    
+
+%%%-----------------------------------------------------------------
+%%% @doc Return the command used to start (this) erlang
+
+-spec get_progname() -> string().
+
+get_progname() ->
+    case init:get_argument(progname) of
+	{ok, [[Prog]]} ->
+	    Prog;
+	_Other ->
+	    "no_prog_name"
```

Comment:
Would an empty string be better here? That string is never matched against even in atom form in OTP.

Reply:
No, the result is used as the base for several command lines passed to os:cmd(), and if empty, the options get interpreted as commands - better with a string that will probably never be a command, so you'll trigger some kind of error.